pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -108,7 +110,12 @@ func newEventConsumer(
 		// Pick a reasonable default.
 		numWorkers = defaultNumWorkers()
 	}
-	if numWorkers <= 1 || isSinkless {
+
+	// We cannot have a separate encoder and sink for parquet format (see
+	// parquet_sink_cloudstorage.go). Because of this the current nprox solution
+	// does not work for parquet format. TODO (ganeshb) Add nprox support for
```

Comment:
nit: let's replace nprox (which is rather internal name) with "Add support for parallel encoding"